### ISSUE TITLE
Executing a function in another database doesn't work

### ISSUE THREAD

**Issue by BorislavZlatanov** [cybertec-postgresql/pg_timetable#617]
I've created two separate databases for timetable and for my app. This is for enhanced security. This issue seems very similar: https://github.com/cybertec-postgresql/pg_timetable/issues/164

Here is the command I use for creating the job:
 `SELECT timetable.add_job('myjob','* * * * *','SELECT public.fetch_data()','["--dbname=my_app_db"]'::jsonb);`

pg_timetable says: `function public.fetch_data() does not exist (SQLSTATE 42883)] [txid:1067] Task execution failed`

But I can verify the function exists in the my_app_db database, in the public schema, and I can successfully execute it from terminal.

This is on Ubuntu 20.04, Postgres 14, pg_timetable 5.6.

**Comment by pashagolub** [comment]
> This issue seems very similar: #164

No, it's absolutely not. That issue if for ancient version of schema. There are even no more such table anymore.

> '["--dbname=my_app_db"]'::jsonb

This is not gonna work. These suppose to be parameters/arguments for tasks, just like function parameters/arguments. 

Check [the sample](https://github.com/cybertec-postgresql/pg_timetable/blob/master/samples/RemoteDB.sql) on how to establish remote task. There is no way to create a remote job with `timetable.add_job()` function
